src/main/java/com/adobe/commerce/cif/model/cart/Cart.java

```diff
@@ -152,14 +156,30 @@ public void setBillingAddress(Address billingAddress) {
         this.billingAddress = billingAddress;
     }
 
+    /**
+     * @deprecated
+     * @return payment Payment.
+     */
     public Payment getPayment() {
         return payment;
     }
 
+    /**
+     * @deprecated
+     * @param payment Payment.
+     */
```

Comment:
You should also add the Java annotation `@Deprecated`

Reply:
@cjelger done.